tests/phpunit/tests/test-wpml-config.php

```diff
@@ -415,6 +415,14 @@ public function test_gutenberg_blocks() {
 				'buttonText' => true,
 			),
 		);
+		$encodings_for_attributes     = array(
+			'my-plugin/my-block'   => array(
+				'other' => 'json,urlencode',
+			),
+			'my-plugin/my-block-7' => array(
+				'other' => 'json,urlencode',
+			),
```

Comment:
Could you add a comment explaining this one will be overriden?

Reply:
Done in [8761bb3](https://github.com/polylang/polylang/pull/1683/commits/8761bb335528fdb15024d67757829a2d02f8c67e).